Dockerfile.template

```diff
@@ -9,22 +9,33 @@ COPY --from=artifacts /srv/repo/*.rpm /rpms/
 COPY --from=mcd /usr/bin/machine-config-daemon /binaries/machine-config-daemon
 
 FROM quay.io/coreos-assembler/fcos:testing-devel
+ARG FEDORA_COREOS_VERSION=412.36.0
+
 COPY . /go/src/github.com/openshift/okd-machine-os
 WORKDIR /go/src/github.com/openshift/okd-machine-os
 COPY --from=rpms /rpms/ /tmp/rpms
 COPY --from=rpms /binaries/machine-config-daemon /usr/libexec/machine-config-daemon
 COPY overlay.d/ /tmp/overlay
-RUN cp -irvf /tmp/overlay/*/* / \
+RUN cat /etc/os-release \ 
+    && rpm-ostree --version \
+    && ostree --version \
+    && cp -irvf /tmp/overlay/*/* / \
     && cp -irvf bootstrap / \
     && cp -irvf manifests / \
+    && rpm-ostree override remove \
+        moby-engine \
+        zincati \
     && rpm-ostree install \
         NetworkManager-ovs \
         open-vm-tools \
         qemu-guest-agent \
+    && rm -f /tmp/rpms/openshift-clients-redistributable-*.rpm \
```

Comment:
Remove the whole `/tmp/rpms`? Not sure if ends up in the layered machine-os-content

Reply:
We need to remove it here before blanket installing all rpms from that dir.
I added `rm -rf /tmp/rpms` further below to remove it during cleanup.